src/test/java/webdriverFramework/SSMission.java

```diff
@@ -1,53 +1,211 @@
 package webdriverFramework;
 
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import enums.Language;
+import com.google.common.base.Function;
+import enums.ValueForDealTypeDropdown;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.openqa.selenium.*;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.remote.server.handler.FindElement;
-import org.openqa.selenium.support.ui.ExpectedCondition;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.Select;
-import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.ui.*;
+import pageObjects.*;
 
 import java.io.File;
+import java.util.*;
+
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
 
-/**
- * Created by roman.pipchenko on 1/20/2017.
- */
 public class SSMission {
-    public static void main(String[] args) {
 
-        File file = new File("C:\\tools\\Firefox\\", "firefox" + ".exe");
-        if (file.exists()) {
-            System.setProperty("webdriver.firefox.bin", "C:\\tools\\Firefox\\firefox.exe");
-        }
+    static WebDriver driver;
+
+    public static String browser = "chrome";
```

Comment:
This looks like you have two different test suites with the same scenario. One suite test on chrome, another on firefox. This might be done better with JUnit suite. You create two suite classes, run which one you need. You'll eliminate a property and an if statement, e.g.

```
@RunWith(Suite.class)
@SuiteClasses({SSMission.class})
public class TestSuiteChrome {°
    @Rule
    public TestRule screenshotTaker = new ScreenshotTaker((TakesScreenshot) driver);

    @BeforeClass
    public static void setUp() {
        System.out.println("setting up Chrome suite...");
 File file = new File("C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe");
    if (file.exists()) {
                System.setProperty("webdriver.chrome.driver", "src\\test\\resources\\chromedriver.exe");
                driver = new ChromeDriver();
            }
    }
}

```

Driver good to have somewhere in static context.